### PR TITLE
Use only Spring, no Java EE!

### PR DESCRIPTION
**Pull request by martin-g**

### CODE REVIEW

File: src/main/java/com/copperarrow/model/dataproviders/UserAccountDataProvider.java [hunk 1]
```diff
@@ -1,48 +1,43 @@
...
 
-import javax.ejb.EJB;
-import javax.ejb.Stateless;
 import java.util.Iterator;
 import java.util.List;
 
+import javax.ejb.Stateless;
+
+import com.copperarrow.dao.UserDAO;
+import com.copperarrow.model.UserAccount;
```
Comment (UlrichKnaack) [inline src/main/java/com/copperarrow/model/dataproviders/UserAccountDataProvider.java]: class is missing

File: src/main/java/com/copperarrow/model/dataproviders/UserAccountDataProvider.java [hunk 2]
```diff
@@ -1,48 +1,43 @@
...
+import org.apache.wicket.spring.injection.annot.SpringBean;
 
-import javax.ejb.EJB;
-import javax.ejb.Stateless;
 import java.util.Iterator;
 import java.util.List;
 
+import javax.ejb.Stateless;
+
+import com.copperarrow.dao.UserDAO;
```
Comment (UlrichKnaack) [inline src/main/java/com/copperarrow/model/dataproviders/UserAccountDataProvider.java:15]: class is missing

File: src/test/java/com/copperarrow/WicketApplicationTest.java [hunk 3]
```diff
@@ -1,15 +1,14 @@
...
 import org.apache.wicket.protocol.http.WebApplication;
 import org.apache.wicket.spring.injection.annot.SpringComponentInjector;
 import org.apache.wicket.spring.test.ApplicationContextMock;
 import org.apache.wicket.util.tester.WicketTester;
 import org.junit.Before;
 import org.springframework.stereotype.Component;
-import org.wicketstuff.javaee.injection.JavaEEComponentInjector;
-import org.wicketstuff.javaee.naming.global.ModuleJndiNamingStrategy;
+
+import com.copperarrow.dao.UserDAO;
```
Comment (UlrichKnaack) [inline src/test/java/com/copperarrow/WicketApplicationTest.java:11]: class is missing

File: src/main/java/com/copperarrow/model/dataproviders/UserAccountDataProvider.java (hunk 1 above)
Comment (UlrichKnaack) [inline src/main/java/com/copperarrow/model/dataproviders/UserAccountDataProvider.java:16]: class is missing

**Comment by dmbeer** [comment]
Hi
@martin-g Thanks for this got me in the right direction meant to close this out some time ago. As I was trying to use JEE.

@UlrichKnaack Thanks for interest if you need this, please feel free to create a new PR so that I can create a new branch.

**Pull request closed without merging**